provision/opentofu/modules/rosa/hcp/scripts/rosa_verify_network.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+if [ -z "$CLUSTER_NAME" ]; then echo "Variable CLUSTER_NAME needs to be set."; exit 1; fi
+
+CLUSTER=$(rosa describe cluster -c ${CLUSTER_NAME} -o json)
+REGION=$(echo ${CLUSTER} | jq -r '.region.id')
+SUBNETS=$(echo ${CLUSTER} | jq -r '.aws.subnet_ids | join(",")')
+rosa verify network --cluster ${CLUSTER_NAME}
+rosa verify network --watch --status-only --region ${REGION} --subnet-ids ${SUBNETS}
```

Comment:
I see that this script spends 3 minutes, and it even spends 3 minutes when I run it now locally. 

To understand why this is necessary, it would be good to have a comment here in this script. Overall, I would hope one could skip it and safe us 3 minutes :-)

Reply:
Yeah this is painful. For unknown reasons the "Inflight Check" fails for the egress when the HCP cluster is first created, however explicitly running the verification again always passes. I need to raise this with the Red Hat terraform team.

Reply:
OK, then please add a comment to the script why it is there, and keep the rest for a follow-up.

Reply:
I asked the Terraform team and they say this is due to the VPC still being initialised when the cluster comes up. They may need to add an additional dependency check for the VPC on the `rosa_hcp_cluster` resource.

In the mean time they say it's possible to add a delay between the VPC and cluster creation. IMO the current solution works well enough and we should wait for this to be rectified in the Terraform provider.